avaframe/in3Utils/initializeProject.py

```diff
@@ -103,22 +105,17 @@ def cleanSingleAvaDir(avaDir, keep=None, deleteOutput=True):
     if deleteOutput:
         _checkForFolderAndDelete(avaDir, 'Outputs')
 
+        # check for *.log files, go to remove only if exists
+        dirToClean = pathlib.Path(avaDir)
+        logFiles = list(dirToClean.glob('*.log'))
+
+        for fi in logFiles:
```

Comment:
so all the logFiles get deleted if deleteOutput?

Reply:
Jup, I don't think it makes much sense to keep the log files when everything they reference is deleted anyways...